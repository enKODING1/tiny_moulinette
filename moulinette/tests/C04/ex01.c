void ft_putstr(char *str);

int main(void)
{
    ft_putstr("");
    ft_putstr("\n");
    ft_putstr("1234\n");
    ft_putstr("Hello 42 Gyeongsan So Hot Place.\n");
    return 0;
}
