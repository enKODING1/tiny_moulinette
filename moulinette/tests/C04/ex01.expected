
1234
Hello 42 Gyeongsan So Hot Place.
